src/Microsoft.Diagnostics.Runtime/src/Common/ModuleInfo.cs

```diff
@@ -115,40 +116,36 @@ public VersionInfo Version
         {
             get
             {
-                if (_version is VersionInfo version)
-                {
-                    return version;
-                }
+                if (_version.HasValue)
+                    return _version.Value;
```

Comment:
Fine with me as well. It's just bad that C# doesn't permit us to treat NVTs and NRTs the same.